scripts/check-theme-lark-imports.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * @license Copyright (c) 2003-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+'use strict';
+
+/* eslint-env node */
+
```

Comment:
I feel like I'm missing a context of why such check is needed. I think it would be good to add a short explanation at the top of the file.

Reply:
Adressed in 76ed693.